tests/manifests/dvImageio.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cdi.kubevirt.io/v1alpha1
+kind: DataVolume
+metadata:
+  name: "test-dv"
+spec:
+  source:
+      imageio:
+         url: "http://f24:8080/ovirt-engine/api"
+         secretRef: "secret"
```

Comment:
Maybe add a fake secret yaml to show the format.

Reply:
The format is the same as the other secrets, so linking to existing secret should be sufficient.

Reply:
Added in #1151